packages/documentation/docs/controls/_toast_styleguide.md

```diff
@@ -0,0 +1,44 @@
+Toasts are small pop-ups that provide simple feedback on a process. They are UI elements where an event causes a small text field to appear at the bottom right of the screen. Toasts are informative, only last a few seconds, and only take up a very small part of the screen to avoid interrupting the workflow. They usually follow an action performed by the user and provide information about the success or failure of that action. We typically use toasts for immediate feedback or tips on actions that a user performs, e.g., successful deletion.
+
+![Overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=2550-58743&t=LITgbzwcgm87dQXa-4)
+
+1. Icon
+2. Button
+3. Timer
+4. Header
+5. Message
+6. Close action
+
+## Options
+
+- **Toast type:** Choose between four preset toast types, or make a custom one: 
+    - Error: Notifies the user that the action cannot be performed due to a specific problem.
+    - Info: Provides the user with additional information about the performed action.
+    - Success: Informs the user of a successfully performed action.
+    - Warning: Warns the user of potential problems that could occur due to the action
+    - Custom: To create your own toast type adjust the icon and its color yourself.
+- **Header:** Add a header for the toast. Use short and concise words. We typically use 1-3 keywords e.g. error occurred. 
+- **Message:** Add a message to explain the toast event in more detail. We typically describe the event precisely and without too many words to keep the sentence as short as possible, e.g., the message was successfully deleted vs. message deleted
```

Comment:
We typically... -> maybe it's possible to shorten a bit, e.g.:

... describe short and precise ...

I would only describe the good version, not the bad one here :)